### ISSUE TITLE
SSO login missing client IP when initiated from a browser.

### ISSUE THREAD

**Issue by AntonAM** [gravitational/teleport#27636]
When client initiate SSO login from the browser we don't pass client login IP stored in the SSO request when creating web session (and we do it when login is initiated from the `tsh`).

**Comment by AntonAM** [comment]
Created related issue (for IdP initiated login): https://github.com/gravitational/teleport/issues/27663

**Comment by wadells** [comment]
We just hit this on platform.teleport.sh when we upgraded from `v12.4.7` to `v13.1.1`.  A previously "working" login configuration locked users out upon upgrade:

```
Failed to create web session.
    IP pinning is enabled for user "walt@goteleport.com" but there is no client IP information
```

We fixed it by disabling ip pinning on the roles with it configured:

https://github.com/gravitational/cloud/pull/4977

Thankfully, this wasn't widely deployed, or we might have locked outselves out of most admin users (since highly privileged users are typically were we'd want to deploy security features like this).

**Comment by AntonAM** [comment]
@wadells yes, unfortunately fix missed `v13.1.1` by a little bit, but it should be present in `v13.1.2`
